src/App/Resources/views/OldProposal/show.html.twig

```diff
@@ -0,0 +1,88 @@
+{% extends '::layout.html.twig' %}
+
+{% block page_content %}
+
+    <h1>{{ oldProposal.oldTitle }}</h1>
+
+    <p>
+        Ancienne version de <a href="{{ path('proposal_show', {'slug': oldProposal.recordedProposal.slug}) }}">{{ oldProposal.recordedProposal.title }}</a>
+    </p>
+
+    {% if oldProposal.recordedProposal.webPath is empty %}
+        <img src="{{ asset('img/science128.png') }}" class="marginAuto">
+    {% else %}
+        <img src="{{ asset(oldProposal.recordedProposal.webPath) }}" class="marginAuto">
+    {% endif %}
+
+    <p>
+        Date de la version : {{ oldProposal.snapDate|date('d/m/Y') }}<br>
+        {% if oldProposal.wasPublished %}
+            Classée dans la thématique : {{ oldProposal.oldTheme.title }}<br>
+        {% else %}
+            Version non-publiée !
+        {% endif %}
+    </p>
+
+    {% if is_granted('not_first_version', oldProposal) %}
+        <a href="{{ path('old_proposal_previous_version', {'slug': oldProposal.slug}) }}">Version précédente</a>
+    {% endif %}
+
+    <a href="{{ path('old_proposal_next_version', {'slug': oldProposal.slug}) }}">Version suivante</a>
+
+    <h2>Auteur de la modification</h2>
+
+    <p>
+        {{ oldProposal.oldAuthor.username }}
+    </p>
+
+    <h2>Auteur</h2>
+
+    <p>
+        {% if oldProposal.oldAuthor is empty %}
+            <p>Cette version n'avait pas d'auteur</p>
+        {% else%}
+            {{ oldProposal.oldAuthor.username }}
+        {% endif %}
+    </p>
+
+    <h2>Soutien(s)</h2>
+
+    <p>
+        {% if oldProposal.oldSupporters is empty %}
```

Comment:
you can directly use :

```twig
{% for user in oldProposal.oldSupporters %}
    foo
{% else %}
    bar
{% endfor %}
```

cf. https://twig.sensiolabs.org/doc/2.x/tags/for.html#the-else-clause